coremltools/converters/mil/frontend/torch/torch_op_registry.py

```diff
@@ -127,3 +129,14 @@ def func_wrapper(func):
         # decorator called without argument
         return func_wrapper
     return func_wrapper(_func)
+
+
+def is_torch_fx_node_supported(torch_fx_node: torch.fx.Node) -> bool:
+    if torch_fx_node.op != "call_function":
+        return False
+
+    torch_fx_node_target_name = torch_fx_node.target.__name__.lower()
+    if torch_fx_node_target_name.startswith("aten."):
```

Comment:
We seem to have multiple name sanitizing logic: this one, [here](https://github.com/apple/coremltools/blob/600fda79b1bdc219a7a07c6fe771835dd08d4f88/coremltools/converters/mil/frontend/torch/internal_graph.py#L188C1-L193C39), [here](https://github.com/apple/coremltools/blob/600fda79b1bdc219a7a07c6fe771835dd08d4f88/coremltools/converters/mil/frontend/torch/internal_graph.py#L188C1-L193C39), and [another one here]. (https://github.com/apple/coremltools/blob/600fda79b1bdc219a7a07c6fe771835dd08d4f88/coremltools/converters/mil/frontend/torch/torch_op_registry.py#L21).

I was wondering if it would be better to unify all this into one helper utility or something similar.

Reply:
+1

Reply:
Good point, filed an [issue](https://github.com/apple/coremltools/issues/2106) to track such naming things, so we'll fix it in a follow-up PR.